lib/utils.js

```diff
@@ -175,6 +256,27 @@ const getMainCodeOwnerfile = async () => {
  */
 const isChangelogLabel = (label) => {
   return label.trim().toUpperCase().startsWith(CHANGELOG_START);
+};
+
+/**
+ * This functions gets the last type of review from a pull request.
```

Comment:
last type of review from a pull request -> last review of a specific type from a pull request